src/components/error_code/ErrorCode.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import './ErrorCode.css';
 import httpResponses from '../../httpResponses';
 
-const ErrorCode = ({code, fetchingMovie, clearSelected}) => {
+const ErrorCode = ({code, clearSelected}) => {
   let response = httpResponses.find(response => response.code === parseInt(code))
   return (
     <article className='error-container'>
       <div className='error-div'>
         <h2 className='error-h2'>Error {response.code}</h2>
         <h3>{response.message}</h3>
         <p>{response.description}</p>
-        {fetchingMovie && <button className='details-back' onClick={clearSelected}>Back</button>}
+        <Link to={'/'}>
```

Comment:
I think these curly brackets can be removed and this can just be `<Link to='/'>`

Reply:
Refactored in new commit